src/content_script.js

```diff
@@ -58,6 +59,13 @@ function isFixed (elem) {
   return false;
 }
 
+function isDisplayNone (elem) {
+  do {
+    if (getComputedStyle(elem).getPropertyValue("display") == "none") return true;
+  } while ((elem = elem.offsetParent));
```

Comment:
Are the extra parens necessary?

Reply:
Yes – The linter burbs without them. I don't understand why, though.